components/tidb_query_codegen/src/rpn_function.rs

```diff
@@ -228,6 +228,9 @@ struct RpnFnAttr {
     /// Whether or not the function is a raw varg function. Raw varg function accepts `&[ScalarValueRef]`.
     is_raw_varg: bool,
 
+    /// Whether or not the function needs extra logic on `None` value.
+    is_nonnull: bool,
```

Comment:
I would prefer `non_null` :)

Reply:
Alright! I'll flip "nullable" for all functions in next PR